tests/console/ansible.pm

```diff
@@ -16,53 +16,64 @@ use warnings;
 use base "consoletest";
 use strict;
 use testapi qw(is_serial_terminal :DEFAULT);
-use serial_terminal qw(select_serial_terminal select_user_serial_terminal);
-use utils qw(zypper_call random_string systemctl file_content_replace);
-use version_utils qw(is_sle is_opensuse is_tumbleweed);
+use serial_terminal 'select_serial_terminal';
+use utils qw(zypper_call random_string systemctl file_content_replace ensure_serialdev_permissions);
+use version_utils qw(is_opensuse is_tumbleweed is_transactional);
 use registration qw(add_suseconnect_product get_addon_fullname);
+use transactional qw(trup_call check_reboot_changes);
 
 sub run {
     select_serial_terminal;
 
     # 1. System setup
 
-    if (is_sle) {
+    unless (is_opensuse) {
         add_suseconnect_product(get_addon_fullname('phub'));
     }
 
+    # Create user account, if image doesn't already contain user
+    # (which is the case for SLE images that were already prepared by openQA)
+    if (script_run("getent passwd $username") != 0) {
+        assert_script_run "useradd -m $testapi::username";
+        assert_script_run "echo '$testapi::username:$testapi::password' | chpasswd";
+    }
+    ensure_serialdev_permissions;
+
     # Install ansible and ansible-test
     # Install python3-yamllint needed for ansible-test
     #   python3-yamllint is available from 15-SP2
     # Install git needed for ansible-galaxy
-    zypper_call 'in ansible git-core python3-yamllint';
+    if (is_transactional) {
+        select_console 'root-console';
+        trup_call('pkg install ansible git-core python3-selinux');
```

Comment:
In the non-transactional case it installs a different set of packages. Why? That deserves a comment or at least adjusting the existing one above

Reply:
I managed to streamline the list. Thank you Fabian!

Reply:
To override existing SSH key.